paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/PaymentMethodRowButtonScreenshotTest.kt

```diff
@@ -0,0 +1,149 @@
+package com.stripe.android.paymentsheet.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.stripe.android.screenshottesting.PaparazziRule
+import com.stripe.android.ui.core.R
+import org.junit.Rule
+import org.junit.Test
+
+internal class PaymentMethodRowButtonScreenshotTest {
```

Comment:
can you add a non-card PM to this screenshot test as well? Not necessary for this PR, but maybe when you add in the styling